src/Microsoft.PowerShell.Commands.Diagnostics/resources/GetEventResources.resx

```diff
@@ -253,7 +253,7 @@ The defined template is following:
     <value>The following value is not in a valid DateTime format: {0}.</value>
   </data>
   <data name="LogInfoUnavailable" xml:space="preserve">
-    <value>Could not retrieve information about the {0} log. Error: {1}.</value>
+    <value>{1} To access the '{0}' log start PowerShell by using the "Run as Administrator" command.</value>
```

Comment:
"Run as Administrator" is a Windows Shell specific thing, may be better to say: ...start PowerShell with elevated user rights

(I searched and we use this type of language already in existing resource strings)

Reply:
Fixed.

We need to review all resx file for "using the "Run as Administrator" command".